lib/migrate/loaders/loaders.js

```diff
@@ -171,10 +201,9 @@ module.exports = function(j, ast) {
 	};
 
 	/**
-	 * Find pre and postLoaders
+	 * Find pre- and postLoaders in ast and put them into the `loaders` array
```

Comment:
Should be "find pre and postLoaders in the ast and put them into the `loaders` array"